source/javascripts/frob_core_helpers.js

```diff
@@ -327,13 +327,22 @@
      * @param {Number} [offset=0] - Additional offset to add to the scrollTop
      */
     smoothScroll: function($target, duration, delay, offset){
+      var _this = this;
+
       duration = this.setDefault(duration, 2000);
       delay = this.setDefault(delay, 100);
       offset = this.setDefault(offset, 0);
 
       setTimeout(function(){
+        var targetOffset = $target.offset().top + offset;
```

Comment:
It was just one of those standards that we set back in the day. I decided that in order to increase readability so that maintenance projects and emergencies that required a rapid kicked-off were less painful, the functions and variables type style separation would be an easy, common convention to follow. 

I regret not doing ALL_CAPS for constants/module-level variables (i.e. `this.MY_MAGIC_PIXEL_OFFSET_NUMBER`, but so it goes.